robot/Cumulus/tests/browser/1_batch_gift_entry/best_match_donation_matching_behaviour.robot

```diff
@@ -26,9 +26,7 @@ Best Match Donation Matching Behaviour
     Select Value From BGE DD    Donor Type    Account
     Wait Until Keyword Succeeds          1 minute
         ...                              5 seconds
-        ...                              Search Field And Wait For Modal    Search Accounts    ${account}[Name]
-
-    Click Link    ${account}[Name]
+        ...                              Search Field And Perform Action   Search Accounts    ${account}[Name]
```

Comment:
Dont think we need Wait Until Keyword Succeds anymore

Reply:
I feel It doesn't harm in anyway. The keyword only tries to repeat the actions if the step fails.  Also was thinking in the lines of additional caution step in the sense, say when we enter the text to search in the lookup field and the dropdown element did not show up for any reason in time, the keyword will try again .